packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -132,13 +132,58 @@ export const css = curry(async function css(
     true
   )
 
+  function getModuleStyleLoader(
+    cssOptions: webpack.ParserOptions,
+    preProcessor: string
+  ) {
+    const loaders: webpack.RuleSetUseItem[] = [
+      // Resolve CSS `@import`s and `url()`s
+      {
+        loader: require.resolve('css-loader'),
+        options: cssOptions,
+      },
+
+      // Compile CSS
+      {
+        loader: require.resolve('postcss-loader'),
+        options: {
+          ident: '__nextjs_postcss',
+          plugins: postCssPlugins,
+          sourceMap: true,
+        },
+      },
+    ].filter(Boolean)
```

Comment:
Why is this filtered?

Reply:
Dead code from: https://github.com/zeit/next.js/pull/10133#discussion_r368719234

Can be removed or will be needed depending on route taken.